netmanager/src/views/components/Activities/ActivitiesToolbar.js

```diff
@@ -0,0 +1,273 @@
+            color="primary"
+            type="submit"
+            align="centre"
+            onClick={() => setOpen(!open)}>
+            {' '}
+            Add Site
+          </Button>
+        </div>
+      </div>
+      <Dialog
+        open={open}
+        onClose={handleSiteClose}
+        aria-labelledby="form-dialog-title"
+        aria-describedby="form-dialog-description">
+        <DialogTitle id="form-dialog-title" style={{ textTransform: 'uppercase' }}>
+          Add a site
+        </DialogTitle>
+
+        <DialogContent>
+          <form className={classes.modelWidth}>
+            <TextField
+              autoFocus
+              margin="dense"
+              label="Site Name"
+              variant="outlined"
+              value={siteData.name}
+              onChange={handleSiteDataChange('name')}
+              fullWidth
+              required
+              error={!!errors.name}
+              helperText={errors.name}
+            />
+            <TextField
+              margin="dense"
+              label="Latitude"
+              variant="outlined"
+              value={siteData.latitude}
+              onChange={handleSiteDataChange('latitude')}
+              error={!!errors.latitude}
+              helperText={errors.latitude}
+              fullWidth
+              required
+            />
+            <TextField
+              margin="dense"
+              label="Longitude"
+              variant="outlined"
+              value={siteData.longitude}
+              onChange={handleSiteDataChange('longitude')}
+              error={!!errors.longitude}
+              helperText={errors.longitude}
+              fullWidth
+              required
+            />
+            <TextField
+              fullWidth
+              margin="dense"
+              label="Network"
+              value={siteData.network}
+              defaultChecked={siteData.network}
+              variant="outlined"
+              error={!!errors.network}
+              helperText={errors.network}
+              InputProps={{
+                classes: {
+                  disabled: useStyles().disabled
+                }
+              }}
+              disabled
+            />
+          </form>
+        </DialogContent>
+
+        <DialogActions>
+          <Grid container alignItems="flex-end" alignContent="flex-end" justify="flex-end">
+            <Button variant="contained" type="button" onClick={handleSiteClose}>
+              Cancel
+            </Button>
+            <Button
+              variant="contained"
+              color="primary"
+              type="submit"
+              onClick={handleSiteSubmit}
+              style={{ margin: '0 15px' }}>
+              Create Site
+            </Button>
+          </Grid>
+          <br />
+        </DialogActions>
+      </Dialog>
+    </>
+  );
+};
+
+ActivitiesToolbar.propTypes = {
+  className: PropTypes.string
+};
+
+export default ActivitiesToolbar;
```

Comment:
@danielmarv what is this code for? I dont see any form in the UI

Reply:
ok @Codebmk

Reply:
This is resolved @Codebmk